### PR TITLE
fix(strategy.class.ts): remove node redirect

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-4261

### Change description ###

Fix the Redirecting on node side.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/common/models/xuiNode.class.ts [hunk 1]
```diff
@@ -38,7 +38,6 @@ export class XuiNode extends EventEmitter {
     public authenticateDefault = (req: Request, res: Response, next: NextFunction): void => {
         if (req.isUnauthenticated()) {
             this.logger.log('unauthenticated, redirecting')
```
Comment (paul-graham) [inline src/common/models/xuiNode.class.ts:40]: And this log message shouldn't say it's redirecting when it's not. 😁

File: src/auth/models/strategy.class.ts [hunk 2]
```diff
@@ -249,7 +249,6 @@ export abstract class Strategy extends events.EventEmitter {
     public authenticate = (req: Request, res: Response, next: NextFunction): void => {
```
[the remark this answers is not in the document]
Comment (RiteshHMCTS) [reply src/auth/models/strategy.class.ts:249]: actually we are putting hotfix hence just removed this redirection; there is some reference to this method but keeping as is for time being

File: src/auth/models/strategy.class.ts [hunk 3]
```diff
@@ -249,7 +249,6 @@ export abstract class Strategy extends events.EventEmitter {
...
         if (req.isUnauthenticated()) {
             this.logger.log('unauthenticated, redirecting')
```
[the remark this answers is not in the document]
Comment (RiteshHMCTS) [reply src/auth/models/strategy.class.ts:251]: same as above

File: src/common/models/xuiNode.class.ts [hunk 4]
```diff
@@ -38,7 +38,6 @@ export class XuiNode extends EventEmitter {
     public authenticateDefault = (req: Request, res: Response, next: NextFunction): void => {
```
[the remark this answers is not in the document]
Comment (RiteshHMCTS) [reply src/common/models/xuiNode.class.ts:38]: same as above

File: src/common/models/xuiNode.class.ts (hunk 1 above)
Comment (RiteshHMCTS) [reply src/common/models/xuiNode.class.ts:40]: same comment

File: src/auth/models/strategy.class.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (paul-graham) [reply src/auth/models/strategy.class.ts:249]: Okay, fair enough.

File: src/auth/models/strategy.class.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (paul-graham) [reply src/auth/models/strategy.class.ts:251]: Can't let you get away with this one, though. The log message is inaccurate.

File: src/common/models/xuiNode.class.ts (hunk 1 above)
Comment (paul-graham) [reply src/common/models/xuiNode.class.ts:40]: The log message is inaccurate and should be changed to reflect that it's no longer redirecting.

Comment (RiteshHMCTS) [reply src/common/models/xuiNode.class.ts:40]: Is it ok if I keep 'unauthenticated', and remove redirecting?

File: src/auth/models/strategy.class.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (RiteshHMCTS) [reply src/auth/models/strategy.class.ts:251]: Is it ok if I keep 'unauthenticated', and remove redirecting?

[the remark this answers is not in the document]
Comment (paul-graham) [reply src/auth/models/strategy.class.ts:251]: Yes, absolutely! 😁

File: src/common/models/xuiNode.class.ts (hunk 1 above)
Comment (paul-graham) [reply src/common/models/xuiNode.class.ts:40]: Yes, absolutely! 😁

File: src/auth/models/strategy.class.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (RiteshHMCTS) [reply src/auth/models/strategy.class.ts]: Updated log message

File: src/common/models/xuiNode.class.ts (hunk 1 above)
Comment (RiteshHMCTS) [reply src/common/models/xuiNode.class.ts]: updated as mentioned

**Review by kuda1992 — APPROVED** [review]
Seems good

**Review by paul-graham — APPROVED** [review]
Thank you for making the requested changes, @RiteshHMCTS, and for answering my questions around the remaining redirection mechanisms. 🙂

**Pull request merged**